typescript/introduction/intro-typescript/what-is-typescript.md

```diff
@@ -0,0 +1,40 @@
+---
+author: kapnobatai136
+
+type: normal
+
+category: must-know
+
+aspects:
+  - introduction
+
+---
+
+# Intro to TypeScript
+
+---
+## Content
+
+Depending on how old you are, you might remember the time JavaScript was just a web development language. Due to its increased versatility, JavaScript is now used in many applications such as designing neural networks, power backends and many others.
+
+You're probably thinking 'Hey, I thought this was a TypeScript course!' yet here we are talking about JavaScript. Because of the high adoption rate, many frameworks (Angular) and libraries (React) were created that aid with development. 
```

Comment:
These two sentences don't flow with one another.

Reply:
Did you mean to move the "Because of the high..." into the end of the paragraph above this one?

Also, there's no need to have the split between framework/library when mentioning Angular or React here. The distinction isn't particularly useful. Especially for learning TypeScript.

 You can use some more neutral less political/pedantic terms like "tool" or "project".